batch/src/main/java/org/frankframework/batch/BatchFileTransformerPipe.java

```diff
@@ -44,6 +45,7 @@
  * @author  John Dekker
  * @deprecated Warning: non-maintained functionality.
  */
+@Deprecated
```

Comment:
Dit hebben wij ooit expres niet gedaan omdat het dan uit de frankdoc verdwijnt en we nog geen alternatief hebben.

Reply:
ohh ok, ik draai het terug!